src/swaps/boltz.rs

```diff
@@ -462,11 +462,74 @@ impl FromStr for RevSwapStates {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(rename_all = "snake_case")]
+pub enum ChainSwapStates {
+    Created,
+    TransactionZeroConfRejected,
+    TransactionMempool,
+    TransactionConfirmed,
+    TransactionServerMempool,
+    TransactionServerConfirmed,
+    TransactionClaimed,
+    TransactionClaimPending,
+    TransactionLockupFailed,
+    SwapExpired,
+    TransactionFailed,
+    TransactionRefunded,
+}
+
+impl ToString for ChainSwapStates {
+    fn to_string(&self) -> String {
+        match self {
+            ChainSwapStates::Created => "swap.created".to_string(),
+            ChainSwapStates::TransactionZeroConfRejected => {
+                "transaction.zeroconf.rejected".to_string()
+            }
+            ChainSwapStates::TransactionMempool => "transaction.mempool".to_string(),
+            ChainSwapStates::TransactionConfirmed => "transaction.confirmed".to_string(),
+            ChainSwapStates::TransactionServerMempool => "transaction.server.mempool".to_string(),
+            ChainSwapStates::TransactionServerConfirmed => {
+                "transaction.server.confirmed".to_string()
+            }
+            ChainSwapStates::TransactionClaimed => "transaction.claimed".to_string(),
+            ChainSwapStates::TransactionClaimPending => "transaction.claim.pending".to_string(),
```

Comment:
`transaction.claim.pending` is for submarine swaps, when we paid the invoice but wait before broadcasting the claim transaction in case you happen to come online to do a key path spend with us

In chain swaps you either:
- key path spend with us and both claim transactions, ours and yours, are broadcasted
- you enforce via script path and we do the same thing

So, that status doesn't need to be here